### ISSUE TITLE
 SyntaxWarning: "is" with a literal. Did you mean "=="?

### ISSUE THREAD

**Issue by XargsUK** [Obliviatum/ObliCogs#11]
/root/.local/share/Red-DiscordBot/data/Lancelot/cogs/CogManager/cogs/matchmaking/matchmaking.py:646: SyntaxWarning: "is" with a literal. Did you mean "=="?  otherkey = 'allowlist' if key is 'denylist' else 'denylist'  

```
		if key in ['allowlist', 'denylist']:
			otherkey = 'allowlist' if key is 'denylist' else 'denylist'
			if isinstance(value, discord.Role):
				skey = 'roles'
			elif isinstance(value, discord.User):
				skey = 'users'
```

**Comment by Obliviatum** [comment]
Yes, "is" must be "==" instead.

**Comment by Obliviatum** [comment]
resolved with commit c0fcc68154553b58857e4970b290ad5343500f18
